ant/easy/node.py

```diff
@@ -120,6 +128,10 @@ def _main(self):
                     self.channels[channel].on_broadcast_data(data)
                 elif data_type == 'burst':
                     self.channels[channel].on_burst_data(data)
+                elif data_type == 'broadcast_tx':                       # Added to support Sending EVENT_TX broadcast messages
+                    self.channels[channel].on_broadcast_tx_data(data)            # Added to support Sending EVENT_TX broadcast messages       
+                elif data_type == 'acknowledge':						# Added to support Rx-Acknowledge
+                    self.channels[channel].on_acknowledge_data(data)	# Added to support Rx-Acknowledge
```

Comment:
Remove comments here

Reply:
Sure! :) expected you like some more comments